### PR TITLE
added sections, more facts, and procedure information

### PR DESCRIPTION
**Pull request by mohamedwane1**

### CODE REVIEW

File: src/main/webapp/statistics.html [hunk 1]
```diff
@@ -7,22 +7,42 @@
...
+      <li>19% of all pregnancies end in abortion</li>
+      <li>1 in 3 American women will have an abortion by age 45</li>
+      <li>6 in 10 women having abortions already have a child</li>
+    </ul>
     <script type="text/javascript">
 
     //load Charts and the corechart package.
     google.charts.load('current', {'packages': ['corechart']});
 
+    //Draw the pie chart  for racial distribution of abortions
```
Comment (sbahaud) [inline src/main/webapp/statistics.html]: For formatting comment, add a space after //. // Draw the pie chart  for racial distribution of abortion
Make the change here and in other comments

File: src/main/webapp/statistics.html [hunk 2]
```diff
@@ -7,22 +7,42 @@
...
-    google.charts.setOnLoadCallback(drawLengthChart);
-
     //draw the ___ chart for types of procedures based on weeks of pregnancy
     //google.charts.setOnLoadCallback(drawProcedureChart);
 
+    function drawRaceChart() {
+      var raceData = google.visualization.arrayToDataTable([
+        ['Ethnic background' , 'Percentage'],
+        ['White',39],
+        ['Black',28],
```
Comment (sbahaud) [inline src/main/webapp/statistics.html]: Add space after comma and not before ['White', 39]

File: src/main/webapp/statistics.html (hunk 1 above)
Comment (mohamedwane1) [reply src/main/webapp/statistics.html]: implemented check

File: src/main/webapp/statistics.html (hunk 2 above)
Comment (mohamedwane1) [reply src/main/webapp/statistics.html]: implemented check

**Review by sbahaud — APPROVED** [review]

**Pull request merged**